.unreleased/LLT-5253

```diff
@@ -0,0 +1 @@
+ci: Bump libtelio-build tag
```

Comment:
do we really want to see it in the changelog? I guess that's the perfect example of a PR with an empty changelog file

Reply:
oh, doesn't make sense then. cleaned that